### ISSUE TITLE
MIOpen 1.8.0 error on gfx906

### ISSUE THREAD

**Issue by kiritigowda** [ROCmSoftwarePlatform/MIOpen#101]
````
OK: OpenVX using GPU device#0 (gfx906+sram-ecc) [OpenCL 2.0 ] [SvmCaps 0 0]
MIOpen(OpenCL): Warning [FindRecordUnsafe] File is unreadable: /opt/rocm/miopen/share/miopen/db/gfx906+sram-ecc_60.cd.pdb.txt
MIOpen(OpenCL): Warning [FindRecordUnsafe] File is unreadable: /opt/rocm/miopen/share/miopen/db/gfx906+sram-ecc_60.cd.pdb.txt
MIOpen(OpenCL): Warning [FindRecordUnsafe] File is unreadable: /opt/rocm/miopen/share/miopen/db/gfx906+sram-ecc_60.cd.pdb.txt
MIOpen(OpenCL): Warning [FindRecordUnsafe] File is unreadable: /opt/rocm/miopen/share/miopen/db/gfx906+sram-ecc_60.cd.pdb.txt
MIOpen(OpenCL): Warning [FindRecordUnsafe] File is unreadable: /opt/rocm/miopen/share/miopen/db/gfx906+sram-ecc_60.cd.pdb.txt
MIOpen(OpenCL): Warning [FindRecordUnsafe] File is unreadable: /opt/rocm/miopen/share/miopen/db/gfx906+sram-ecc_60.cd.pdb.txt
'gfx906+sram-ecc' is not a recognized processor for this target (ignoring processor)
'gfx906+sram-ecc' is not a recognized processor for this target (ignoring processor)
<stdin>:464:5: error: Only Gfx8 and Gfx9 ISA is supported
    .error "Only Gfx8 and Gfx9 ISA is supported"
    ^
<stdin>:1570:1: error: unmatched .ifs or .elses

^
MIOpen(OpenCL): Warning [AmdgcnAssemble]  -x assembler -target amdgcn--amdhsa -mno-code-object-v3 -mcpu=gfx906+sram-ecc  -Wa,-defsym,stride_h=1 -Wa,-defsym,stride_w=1 -Wa,-defsym,img_h=56 -Wa,-defsym,img_w=56 -Wa,-defsym,batch_size=2 -Wa,-defsym,input_channels=64 -Wa,-defsym,output_channels=64 -Wa,-defsym,wei_h=1 -Wa,-defsym,wei_w=1 -Wa,-defsym,pad_h=0 -Wa,-defsym,pad_w=0 -Wa,-defsym,weights_layout=0 -Wa,-defsym,vec_c_in=1 -Wa,-defsym,vec_k_out=1 -Wa,-defsym,vec_c_filter=1 -Wa,-defsym,acc_type=1 -Wa,-defsym,buf_type=1 -Wa,-defsym,input_n_stride=802816 -Wa,-defsym,input_c_stride=12544 -Wa,-defsym,input_h_stride=224 -Wa,-defsym,input_w_stride=4 -Wa,-defsym,output_n_stride=802816 -Wa,-defsym,output_k_stride=12544 -Wa,-defsym,output_h_stride=224 -Wa,-defsym,output_w_stride=4 -Wa,-defsym,filter_k_stride=256 -Wa,-defsym,filter_c_stride=4 -Wa,-defsym,filter_h_stride=4 -Wa,-defsym,filter_w_stride=4 -Wa,-defsym,input_buffer_size=1605632 -Wa,-defsym,filter_buffer_size=16384 -Wa,-defsym,output_buffer_size=1605632 -Wa,-defsym,ROCM_METADATA_VERSION=4 -Wa,-defsym,read_size=1 -Wa,-defsym,k_mult=4 -Wa,-defsym,chunks_per_wave=4 -Wa,-defsym,chunk_size=1 -Wa,-defsym,n_mult=1 -Wa,-defsym,c_mult=1 -Wa,-defsym,waves_c_in_group=1 -Wa,-defsym,waves_k_in_group=1 - -o /tmp/miopen-tmp-7634-754a-82a5-46da/amdgcn-asm-out-XXXXXX
MIOpen Error: /root/deps/MIOpen-1.8.0/src/ocl/gcn_asm_utils.cpp:228: Assembly error(1)

````

**Comment by daniellowell** [comment]
Can you try building develop from our private repo? 
Also, can you run your test using the environmental variable: MIOPEN_LOG_LEVEL=6
Then attach the output, or send it to me?

**Comment by daniellowell** [comment]
Fixed in 1.8.1.
